### PR TITLE
PHPCS: Autofix the "class" files within the Jetpack root

### PR DESCRIPTION
**Pull request by kraftbj**
A lot of PHPCBF action.

#### Changes proposed in this Pull Request:
* To help developers use built-in PHPCS tools without having to see a bunch of auto-fixable things.

#### Testing instructions:
* Generally use Jetpack but read the changes carefully.

#### Proposed changelog entry for your changes:
*

### CODE REVIEW

File: class.jetpack.php [hunk 1]
```diff
@@ -6547,7 +6673,7 @@ function concat_remove_style_loader_tag( $tag, $handle ) {
 		if ( in_array( $handle, $this->concatenated_style_handles ) ) {
 			$tag = '';
 			if ( defined( 'WP_DEBUG' ) && WP_DEBUG ) {
-				$tag = "<!-- `" . esc_html( $handle ) . "` is included in the concatenated jetpack.css -->\r\n";
+				$tag = '<!-- `' . esc_html( $handle ) . "` is included in the concatenated jetpack.css -->\r\n";
```
Comment (jeherve) [inline class.jetpack.php]: Not sure why one gets updated but not the other. Because of the carriage return?

**Review by jeherve — CHANGES_REQUESTED** [review]
This is mostly okay, I just have a few minor comments.

File: class.jetpack.php (hunk 1 above)
Comment (kraftbj) [reply class.jetpack.php]: Correct, need `""` for the carriage returns to be processed as such and not literally as the string `\r\n`.

File: class.jetpack.php [hunk 2]
```diff
@@ -1478,11 +1503,12 @@ public static function get_avatar_url( $id_or_email, $size = 96, $default = '',
 	 * array (
 	 *      'plugins'                       => (int) Number of plugin updates available.
 	 *      'themes'                        => (int) Number of theme updates available.
-	 *      'wordpress'                     => (int) Number of WordPress core updates available.
+	 *      'WordPress'                     => (int) Number of WordPress core updates available. // phpcs:ignore WordPress.WP.CapitalPDangit
```
Comment (jeherve) [inline class.jetpack.php]: It looks like WordPress is still capitalized.

```suggestion
	 *      'wordpress'                     => (int) Number of WordPress core updates available. // phpcs:ignore WordPress.WP.CapitalPDangit.Misspelled
```

**Review by jeherve — CHANGES_REQUESTED** [review]

**Review by jeherve — APPROVED** [review]
This looks good now. I'll let you merge, and take care of Fusion sync :)

**Comment by kraftbj** [comment]
r196688-wpcom and r196689-wpcom

**Pull request merged**